### PR TITLE
news/front-end-modules (Task No. 40)

### PR DESCRIPTION
**Pull request by netzarbeiter**

### CODE REVIEW

File: docs/manual/core-extensions/faq/frontend-modules.en.md [hunk 1]
```diff
@@ -44,32 +47,43 @@ The front-end module "FAQ list" adds a list of questions to the website, which c
...
+
 
 ### Comments settings
 
 **Comment template:** Here you can select the comment template.
 
-**HTML outputThe**  
- frontend module generates **the** following HTML code:
+**The HTML output**  
+The frontend module generates the following HTML code:
```
Comment (xchs) [inline docs/manual/core-extensions/faq/frontend-modules.en.md]:
```suggestion
The front end module generates the following HTML code:
```

File: docs/manual/core-extensions/faq/frontend-modules.en.md [hunk 2]
```diff
@@ -91,20 +105,25 @@ On a single page there may only be one "reader module" at a time, regardless of
...
 
 ### Template settings
 
-**Individual template:** Here you can overwrite the default template.
+**Module template:** Here you can overwrite the default template.
 
-**HTML outputThe**  
- frontend module generates the following HTML code:
+**The HTML output**  
+The frontend module generates the following HTML code:
```
Comment (xchs) [inline docs/manual/core-extensions/faq/frontend-modules.en.md]:
```suggestion
The front end module generates the following HTML code:
```

File: docs/manual/core-extensions/news/frontend-modules.en.md [hunk 3]
```diff
@@ -1,79 +1,92 @@
...
-The frontend module "message list" displays any number of posts from one or more message archives in the frontend. Which parts of a news item are displayed depends on the respective template. Everything is possible from a simple headline to a complete post.
+The front end module "message list" displays any number of posts from one or more message archives in the front end. 
+Which parts of a news item are displayed depends on the respective template. Everything is possible from a simple 
+headline to a complete post.
+
 
 ### Module configuration
 
-**News archives:** Here you define from which archives posts should be listed. By default the posts are sorted by date in descending order.
+**News archives:** Here you define from which archives posts should be listed. By default the posts are sorted by date
```
Comment (xchs) [inline docs/manual/core-extensions/news/frontend-modules.en.md]:
```suggestion
**News archives:** Here you define from which archives posts should be listed. By default, the posts are sorted by date 
```

File: docs/manual/core-extensions/news/frontend-modules.en.md [hunk 4]
```diff
@@ -1,79 +1,92 @@
...
-**Skip elements:** Here you define the number of items to be skipped.
+**Items per page:** If you enter a value greater than 0, Contao automatically distributes the posts to multiple pages - 
+assuming you have enough of them.
 
-**Elements per page:** If you enter a value greater than 0, Contao automatically distributes the posts to multiple pages - assuming you have enough of them.
 
 ### Template settings
 
-**Meta fields:** Here you can specify which meta-information (date of the post, author of the post and number of comments) is displayed.
+**Meta fields:** Here you can specify which meta-information (date of the post, author of the post and number of
```
Comment (xchs) [inline docs/manual/core-extensions/news/frontend-modules.en.md]:
```suggestion
**Meta fields:** Here you can specify which meta information (date of the post, author of the post and number of 
```

File: docs/manual/core-extensions/news/frontend-modules.en.md [hunk 5]
```diff
@@ -88,50 +101,63 @@ Caution**:** in most cases this functionality should not be used for message lis
...
-The front-end module "news reader" is used to display a specific news item. The module obtains the alias of the article via the URL, so that news with so-called permanent links can be specifically linked:
+The front end module "news reader" is used to display a specific news item. The module obtains the alias of the article 
+via the URL, so that news with so-called permanent links can be specifically linked:
 
-`www.domain.de/nachricht/form-folgt-funktion.html`
+`www.example.com/news/form-follows-function.html`
 
-In this example, the message with the alias "form-folgt-funktion" is called via the "message" page: If the searched message does not exist, the message reader returns an error message and the HTTP status code "404 Notfound". The status code is important for search engine optimization.
+In this example, the message with the alias "form-follows-function" is called via the "news" page: If the searched 
+message does not exist, the message reader returns an error message and the HTTP status code "404 Notfound". The
```
Comment (xchs) [inline docs/manual/core-extensions/news/frontend-modules.en.md]:
```suggestion
message does not exist, the message reader returns an error message and the HTTP status code "404 Not Found". The 
```

File: docs/manual/core-extensions/news/frontend-modules.en.md [hunk 6]
```diff
@@ -88,50 +101,63 @@ Caution**:** in most cases this functionality should not be used for message lis
...
+
 
 ### Template settings
 
-**Meta-fields:** Here you can define which meta information (date of the post, author of the post and number of comments) is displayed.
+**Meta fields:** Here you can define which meta information (date of the post, author of the post and number of 
+comments) is displayed.
 
-**Message template:** Here you select the message template. By `news_full`default, the template displays the complete article.
+**News template:** Here you select the message template. By default `news_full`, the template displays the complete
```
Comment (xchs) [inline docs/manual/core-extensions/news/frontend-modules.en.md]:
```suggestion
**News template:** Here you select the message template. By default, the template `news_full` displays the complete 
```

File: docs/manual/core-extensions/news/frontend-modules.en.md [hunk 7]
```diff
@@ -208,32 +244,39 @@ The front-end module "News archive" is used to list all news items of a certain
 <!-- indexer::continue -->
 ```
 
-## Message archive menu {#Message archive-menu}
 
-The frontend module "News Archive Menu" adds a menu to the website, which allows you to view the posts of the individual days, months or years.
+## Message archive menu
```
Comment (xchs) [inline docs/manual/core-extensions/news/frontend-modules.en.md]:
```suggestion
## News archive menu
```

File: docs/manual/core-extensions/faq/frontend-modules.en.md [hunk 8]
```diff
@@ -1,35 +1,38 @@
...
 aliases:
-    - /en/core-extensions/faq/frontend-modules/
+    - /en/core-extensions/faq/front-end-modules/
 weight: 20
 ---
 
-{{% notice warning %}}
-This article is machine translated.
-{{% /notice %}}
+Now that you know how categories and questions are managed in the backend, we will explain how to display this content
```
Comment (xchs) [inline docs/manual/core-extensions/faq/frontend-modules.en.md]:
```suggestion
Now that you know how categories and questions are managed in the back end, we will explain how to display this content 
```

File: docs/manual/core-extensions/faq/frontend-modules.en.md [hunk 9]
```diff
@@ -44,32 +47,43 @@ The front-end module "FAQ list" adds a list of questions to the website, which c
...
-The frontend module "FAQ reader" is used to display the answer to a specific question. The module obtains the alias of the question via the URL, so that FAQs can be linked with so-called [permalinks](https://de.wikipedia.org/wiki/Permalink):
+The front end module "FAQ reader" is used to display the answer to a specific question. The module obtains the alias of 
+the question via the URL, so that FAQs can be linked with so-called [permalinks](https://en.wikipedia.org/wiki/Permalink):
 
-`example.com/frage/kann-ich-eigene-php-skripte-verwen-den.html`
+`example.com/question/can-I-use-my-own-php-scripts.html`
 
-The keyword of the FAQ reader is *frage fra*ge and tells the module to search and output a specific question. If the searched question does not exist, the FAQ reader returns an error message and the HTTP status code "404 Not found". The status code is important for search engine optimization.
+The keyword of the FAQ reader is **question** and tells the module to search and output a specific question. If the 
+searched question does not exist, the FAQ reader returns an error message and the HTTP status code "404 Not found".
```
Comment (xchs) [inline docs/manual/core-extensions/faq/frontend-modules.en.md]:
```suggestion
searched question does not exist, the FAQ reader returns an error message and the HTTP status code "404 Not Found". 
```

**Review by xchs — COMMENTED** [review]
👍🏻

**Comment by netzarbeiter** [comment]
Thank you, Hannes

**Pull request merged**